src/utils/datetime/formatAsRelative.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * @flow
+ */
+
+import { DateTime, Interval } from 'luxon';
+
+import formatDateTime from './formatDateTime';
+import { INVALID_DATE_TIME, RELATIVE_TIMES } from './constants';
+
+export default function formatAsRelative(
+  value :string = '',
+  format :Intl$DateTimeFormatOptions = DateTime.DATE_SHORT, // luxon doesn't export IntlDateTimeFormatOptions yet
+  fallback :string = INVALID_DATE_TIME
+) :string {
+  let relativeDate = formatDateTime(value, format, fallback);
+  const valueDT = DateTime.fromISO(value);
+  const today = DateTime.local();
+  const yesterday = DateTime.local().minus({ days: 1 });
+  const oneWeekAgo = DateTime.local().minus({ weeks: 1 });
+  const oneMonthAgo = DateTime.local().minus({ months: 1 });
+  const oneYearAgo = DateTime.local().minus({ years: 1 });
```

Comment:
No need to call `DateTime.local()` every time because you already have `today`.

```
const today = DateTime.local();
const yesterday = today.minus({ days: 1 });
const oneWeekAgo = today.minus({ weeks: 1 });
const oneMonthAgo = today.minus({ months: 1 });
const oneYearAgo = today.minus({ years: 1 });
```